handout/a11y/key-concerns/visual-assistance.md

```diff
@@ -0,0 +1,9 @@
+# Visual Assistance
+
+## Color Contrast
+
+One category of disability are people åwho are color blind or deficient. This means that when choosing colors to represent different elements on the website, the contrast between them needs to be considered. For WCAG 2.0 AA, this means that the contrast ratio for text or visual representations of text needs to be at least [4.5:1](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast). The contrast ratio of two different colors is not really obvious so there are tools online to measure the contrast ratio such as this [color contrast checker](http://webaim.org/resources/contrastchecker/) from WebAIM or be checked with using automation tests.
```

Comment:
Regarding automated tests, I was planning on referencing or linking the automated testing section of the course once that's added. I like the idea of providing an example like the one in the link